osquery/tables/system/windows/intel_me.cpp

```diff
@@ -0,0 +1,194 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <vector>
+
+#define WIN32_LEAN_AND_MEAN
+// clang-format off
+#include <Windows.h>
+#include <SetupAPI.h>
+// clang-format on
+#include <initguid.h>
+#include <tchar.h>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/logger.h>
+
+namespace osquery {
+namespace tables {
+
+DEFINE_GUID(HECI_INTERFACE_GUID,
+            0xE2D1FF34,
+            0x3458,
+            0x49A9,
+            0x88,
+            0xDA,
+            0x8E,
+            0x69,
+            0x15,
+            0xCE,
+            0x9B,
+            0xE5);
+
+struct me_version {
+  uint8_t major;
+  uint8_t minor;
+  uint16_t build;
```

Comment:
As discussed, maybe flip build with hotfix? And flip the order below, so major.minor.hotfix.build?